deploy/script/katello-service

```diff
@@ -20,13 +20,9 @@ if [ $EUID -ne 0 ]; then
   exit 1
 fi
 
-if [ -x /etc/init.d/tomcat6 -o -x /lib/systemd/system/tomcat6.service ]; then
-  TOMCAT="tomcat6"
-fi
+# Reversed order so qpid comes before tomcat - BZ1147263
+SERVICES=$(/bin/ls -r /etc/katello/services.d)
```

Comment:
It's a temporary workaround, you really should be able to restart services in any order, and can once that BZ is resolved